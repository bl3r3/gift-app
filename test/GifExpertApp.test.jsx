import { render, screen } from "@testing-library/react";
import { GifExpertApp } from "../src/GifExpertApp";

describe("Test en <GifExpertApp />", () => {
  it("Match to snapshot", () => {
    const { container } = render(<GifExpertApp />);
    expect(container).toMatchSnapshot();
  });

  it("Render Title app", () => {
    render(<GifExpertApp />);
    expect(screen.getByText("GifExpertApp"));
  });
});
