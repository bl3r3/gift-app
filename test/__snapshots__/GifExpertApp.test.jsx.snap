// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test en <GifExpertApp /> Match to snapshot 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="buscar gifs"
      type="text"
      value=""
    />
  </form>
  <h2>
    Loading...
  </h2>
  <h3>
    one punch
  </h3>
  <div
    class="card-grid"
  />
</div>
`;
